rfp-proposals/filexplorer.org.md

```diff
@@ -0,0 +1,180 @@
+	* Sub-repo for documentation and milestone tracking
+* Well-documented codebase hosted on SSD2 instance using [MkDocs](https://www.mkdocs.org/).
+* User-facing documentation (hosted on separate subdomain running MkDocs), and contributing guidelines + good first issues to promote future open-source contributions from the community.
+
+## Development Roadmap
+
+The following milestones describe the development outline of filexplorer.org. Please note: the number of people working on each milestone has been omitted since this RFP response is being delivered individually.
+
+| Phase | Milestone Description | Funding | Timeline |
+|-------|-----------------------|---------|----------|
+|      1 | Setup, Design, Front-end framing                     |        $750 |   3 weeks (October 1st - 21st)       |
+|      2 | Back-end, MongoDB, Completed Implementation                      |       $1500  |   8 weeks (October 22nd - December 16)       |
+|      3 |   Documentation, API Guides, Addition to Shipyard                    |       $500  |   1 week (December 17 - 21, excluding winter festivities)       |
+|      4 |  Long-term maintenance and support                     |       $750  |  9 months (January - September 2020)        |
+
+
+#### Phase 1
+The first phase involves a few things. This involves multiple sub-milestones:
+
+1. Procuring domain name and hosting (setting up load balancing, static asset CDN, SSL certificate via Let's Encrypt)
+2. Setting up Github organization, appropriate licensing for each sub-repo, and integration with Github Actions to automatically deploy to staging and development environments on commit.
+3. High-fidelity design mock-ups in Adobe XD. These will include finalized top level summary page mock-ups, and somewhat finalized secondary page mock-ups. Secondary page mock-ups are not set in stone since I'd like to iterate and continue to add upon the displayed metrics and their styling as I work.
+4. Work-in-progress staging site deployed with:
+	1. Working basic data retrieval via back-end and MongoDB.
+	2. Fully implemented, responsive, header, footer, and CSS framework.
+	3. Completed page sitemap.
+
+For the time being, I am planning on developing with a Preact front-end, which is a highly optimized and minified variant of React. The end goal is to deliver a performant platform, which Preact will help in doing.
+
+#### Phase 2
+Phase 2 is the largest portion of the project since it involves:
+1. Back-end completion.
+	1. During Phase 2, the back-end must be completed up to spec in Express with completed and deployed endpoints in use by the front-end. In specific, the following metrics are being targeted for the time being, although more may be added as development progresses: 
+		1. Top level:
+			1. Block stats
+				1. Tipset Height
+				2. Number of blocks in tipset
+				3. Hash of parent block in tipset
+				4. Avg. block time
+				5. Last block miner address & PeerID
```

Comment:
Is this also referring to information about tipsets? Is this the miner address/peer ID for each block in a tipset?